include/proxy/v4/proxy_fmt.h

```diff
@@ -11,7 +11,12 @@
 #error Please ensure that proxy.h is included before proxy_fmt.h.
 #endif // __msft_lib_proxy4
 
-#if FMT_VERSION >= 60100
+#if FMT_VERSION >= 120000
```

Comment:
Please revert this change. Proxy should not rely on `fmt`'s implementation details for future builds, and the existing `static_assert` already provides sufficient diagnostics.